### PR TITLE
drop jQuery UI autocomplete

### PR DESCRIPTION
**Pull request by orangejenny**
depends on https://github.com/dimagi/commcare-hq/pull/13731, https://github.com/dimagi/commcare-hq/pull/13709, https://github.com/dimagi/commcare-hq/pull/13708, and https://github.com/dimagi/commcare-hq/pull/13697

@calellowitz

### CODE REVIEW

**Comment by benrudolph** [comment]
👀 nice

**Comment by orangejenny** [comment]
also depends on https://github.com/dimagi/commcare-hq/pull/13807

**Comment by orangejenny** [comment]
ready to go, everything dependent on this has been merged
new code buddy @dannyroberts

File: corehq/apps/style/templates/style/base.html [hunk 1]
```diff
@@ -167,8 +167,6 @@
             <script src="{% static 'jquery-ui/ui/position.js' %}"></script>
 
             <!-- Individual widgets and interactions -->
-            <script src="{% static 'jquery-ui/ui/menu.js' %}"></script>
-            <script src="{% static 'jquery-ui/ui/autocomplete.js' %}"></script>
```
Comment (dannyroberts) [inline corehq/apps/style/templates/style/base.html]: can we remove this from bower.json as well?

Comment (orangejenny) [inline corehq/apps/style/templates/style/base.html]: No, bower pulls in all of jquery ui, and we're still using other widgets.

**Pull request merged**